manifests/g/GAI/GAI/1.0.0/GAI.GAI.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GAI.GAI
+PackageVersion: 1.0.0
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{84231572-4192-4BA0-BB22-C239D1B8ABFB}_is1'
+ReleaseDate: 2024-12-22
+AppsAndFeaturesEntries:
+- Publisher: Heshan Aisipai E-commerce Co., Ltd.
+  ProductCode: '{84231572-4192-4BA0-BB22-C239D1B8ABFB}_is1'
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\Gai'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://webpath.iche2.com/release/Gai-1.0.0-Setup.exe
+  InstallerSha256: 7BE0AD53EDA3E6A08F70843EB068B1C6FBA880C7A33672A98192045869C1E856
```

Comment:
```suggestion
  InstallerSha256: 7B2AD08A6EF5B926407CEA8147C4C80C0C32718C17378B24CA1C779A6D335F13
```

Reply:
My download page has a SHA256 checksum of 7B2AD08A6EF5B926407CEA8147C4C80C0C32718C17378B24CA1C779A6D335F13.